client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx

```diff
@@ -5,52 +5,56 @@ import { Collapse, Grid, Typography, Divider, IconButton } from '@material-ui/co
     
 import Contact from 'models/Contact';
 import Toggle from 'commons/Toggle/Toggle';
-import { initAddress } from 'models/Address';
 import useFormStyles from 'styles/formStyles';
 import { timeFormat } from 'Utils/displayUtils';
 import DatePick from 'commons/DatePick/DatePick';
 import FormInput from 'commons/FormInput/FormInput';
 import InteractionEventDialogData from 'models/Contexts/InteractionEventDialogData';
-import LocationsTypesAndSubTypes from 'commons/Forms/LocationsTypesAndSubTypes/LocationsTypesAndSubTypes';
+import PlacesTypesAndSubTypes from 'commons/Forms/PlacesTypesAndSubTypes/PlacesTypesAndSubTypes';
 
 import ContactForm from './ContactForm/ContactForm';
 import useStyles from './InteractionEventFormStyles';
 import OfficeEventForm from '../InteractionEventForm/PlacesAdditionalForms/OfficeEventForm';
-import SchoolEventForm, { grades } from '../InteractionEventForm/PlacesAdditionalForms/SchoolEventForm';
-import HospitalEventForm, { hospitals } from '../InteractionEventForm/PlacesAdditionalForms/HospitalEventForm';
+import SchoolEventForm from '../InteractionEventForm/PlacesAdditionalForms/SchoolEventForm';
+import { hospitals } from '../InteractionEventForm/PlacesAdditionalForms/HospitalEventForm';
 import DefaultPlaceEventForm from '../InteractionEventForm/PlacesAdditionalForms/DefaultPlaceEventForm';
 import PrivateHouseEventForm from '../InteractionEventForm/PlacesAdditionalForms/PrivateHouseEventForm';
-import TransportationEventForm, { resetTransportationFormFields } from '../InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/TransportationEventForm';
-import { InteractionEventDialogContext } from '../InteractionsEventDialogContext/InteractionsEventDialogContext';
+import TransportationEventForm from '../InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/TransportationEventForm';
+import { InteractionEventDialogContext, initialDialogData } from '../InteractionsEventDialogContext/InteractionsEventDialogContext';
+import OtherPublicLocationForm from './PlacesAdditionalForms/OtherPublicLocationForm';
+import MedicalLocationForm, { hospitalPlaceType } from './PlacesAdditionalForms/MedicalLocationForm';
 
-const privateHouseLocationType : string = 'בית פרטי';
-const officeLocationType : string = 'משרד';
-export const transportationLocationType : string = 'תחבורה';
-export const schoolLocationType : string = 'בית ספר';
-export const hospitalLocationType : string = 'בית חולים';
+const privateHousePlaceType : string = 'בית פרטי';
+const officePlaceType : string = 'משרד';
+const transportationPlaceType : string = 'תחבורה';
+const schoolPlaceType : string = 'מוסד חינוכי';
+const medicalPlaceType : string = 'מוסד רפואי';
+const religionPlaceType : string = 'אתר דת';
+const geriatricPlaceType : string = 'מוסד גריאטרי';
+const otherPublicPlaceType : string = 'מקומות ציבוריים נוספים';
```

Comment:
this consts declared additionally or instead of getting the place types from the DB?

Reply:
Additionally

Reply:
ok